### PR TITLE
add legend on form

### PR DESCRIPTION
**Pull request by SOLenG**

### CODE REVIEW

File: BackofficeBundle/Resources/views/Form/form_div_layout.html.twig [hunk 1]
```diff
@@ -190,3 +191,11 @@
         {% endif %}
     {% endspaceless %}
 {% endblock oo_date_picker_widget %}
+{% block form_legend %}
+    <div class="form-group form-legend">
```
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/views/Form/form_div_layout.html.twig]: class form-legend is never used why add it ? and i think than would be a good idea to use the class help-block of bootstrap

File: BackofficeBundle/Resources/views/Form/form_div_layout.html.twig [hunk 2]
```diff
@@ -190,3 +191,11 @@
...
+        <div class="col-lg-12">
+            <p>* {{ 'open_orchestra_backoffice.form.orchestra_fields.required_field' | trans }}</p>
+            <p>** {{ 'open_orchestra_backoffice.form.node.template_selection.helper' | trans }}</p>
```
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/views/Form/form_div_layout.html.twig]: we wern't told that we added help text for ** only on the forms which has ?

**Comment by SOLenG** [comment]
it's updated @itkg-alavieille

File: BackofficeBundle/Resources/views/Form/form_div_layout.html.twig [hunk 3]
```diff
@@ -190,3 +195,11 @@
         {% endif %}
     {% endspaceless %}
 {% endblock oo_date_picker_widget %}
+{% block form_legend %}
+    <div class="help-block">
+        <p>* {{ 'open_orchestra_backoffice.form.orchestra_fields.required_field' | trans }}</p>
+        {% if template_selection_helper is defined and template_selection_helper == true %}
+            <p>** {{ 'open_orchestra_backoffice.form.node.template_selection.helper' | trans }}</p>
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/views/Form/form_div_layout.html.twig]: Where is defined this translation key ? Does it require a PR to add it ?

Comment (SOLenG) [inline BackofficeBundle/Resources/views/Form/form_div_layout.html.twig]: @itkg-ngilain I used the translations already present in the "messages.x" file on the BO bundle

Comment (itkg-ngilain) [inline BackofficeBundle/Resources/views/Form/form_div_layout.html.twig]: ok

**Comment by itkg-ngilain** [comment]
:+1:

File: BackofficeBundle/Resources/views/Form/form_div_layout.html.twig [hunk 4]
```diff
@@ -190,3 +195,11 @@
         {% endif %}
     {% endspaceless %}
 {% endblock oo_date_picker_widget %}
+{% block form_legend %}
+    <div class="help-block">
+        <p>* {{ 'open_orchestra_backoffice.form.orchestra_fields.required_field' | trans }}</p>
+        {% if template_selection_helper is defined and template_selection_helper == true %}
```
Comment (nicolasThal) [inline BackofficeBundle/Resources/views/Form/form_div_layout.html.twig]: I would put a variable name which is more generic, this variable is only linked to the template selection and the ** is for much more

**Comment by SOLenG** [comment]
it's updated @nicolasThal

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**